sportnet/sportnet/authapp/views.py

```diff
@@ -1,3 +1,26 @@
-from django.shortcuts import render
+from __future__ import unicode_literals
 
-# Create your views here.
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.http import HttpResponseRedirect
+from django.urls import reverse_lazy
+
+
+from django.views.generic import FormView
+
+from .forms import UserLoginForm
+
+User = get_user_model()
+
+
+class LoginView(FormView):
+    form_class = UserLoginForm
+    template_name = 'registration/login.html'
```

Comment:
Шаблон лучше складывать в папку с именем приложения, в данном случае 'authappp' (вместо 'registration'). Грубо говоря, просто переименовать в шаблонах 'registration' в 'authapp'
```suggestion
    template_name = 'authapp/login.html'
```